Doc/whatsnew/3.8.rst

```diff
@@ -883,6 +883,10 @@ The following features and APIs have been removed from Python 3.8:
   :func:`fileinput.FileInput` which was ignored and deprecated since Python 3.6
   has been removed. :issue:`36952` (Contributed by Matthias Bussonnier)
 
+* The function :func:`sys.set_coroutine_wrapper` deprecated in Python 3.7 has
+  been removed; :func:`sys.get_coroutine_wrapper` now always return ``None``.
```

Comment:
I don't quite get it -- what's the point of keeping `sys.get_coroutine_wrapper`? Both should be removed.

Reply:
> I don't quite get it -- what's the point of keeping sys.get_coroutine_wrapper? Both should be removed.

I remember reading to remove sys.set_coroutine_wrapper now and get_coro_wrapper later... but my memory might just be failing me. I'm happy to remove get_coro_wrapper as well.

Reply:
Done, and rebased (Conflict, need to regen clinic).